### PR TITLE
Fixes #26812,#26814 - display correct vm_details in notifications

### PR DESCRIPTION
**Pull request by kgaikwad**
Note that - state method is renamed to `status` in fog-google.

### CODE REVIEW

File: app/views/compute_resources_vms/show/_gce.html.erb [hunk 1]
```diff
@@ -5,28 +5,16 @@
...
     </thead>
     <tbody>
+      <%= prop :state %>
+      <%= prop :pretty_machine_type, "Machine Type" %>
+      <%= prop :zone_name, "Zone" %>
       <tr>
-        <td>State</td>
-        <td><%= @vm.status.downcase %></td>
+        <td>Created</td>
+        <td><%= Time.parse(@vm.creation_timestamp).utc %></td>
```
Comment (tbrisker) [inline app/views/compute_resources_vms/show/_gce.html.erb]: Should this use `date_time_relative_value` helper for consistency with other time fields?

**Review by tbrisker — CHANGES_REQUESTED** [review]
Small comment inline, otherwise lgtm

**Review by ezr-ondrej — DISMISSED** [review]

**Comment by ezr-ondrej** [comment]
LGTM :+1:

**Comment by kgaikwad** [comment]
Thank you @tbrisker and @ezr-ondrej. I have updated PR with suggested changes.

**Review by tbrisker — APPROVED** [review]
Thanks @kgaikwad !

**Comment by tbrisker** [comment]
1.22 - ec5d2f1eb a74dc7b17

**Pull request merged**